lib/constants/config.dart

```diff
@@ -2,13 +2,19 @@ class Config {
   static final defaultEndpoint = "https://api.telosfoundation.io";
   static final hyphaEndpoint = "https://node.hypha.earth";
   static final onboardingAccountName = "join.seeds";
-  static final onboardingPrivateKey = "5JhM4vypLzLdDtHo67TR5RtmsYm2mr8F2ugqcrCzfrMPLvo8cQW";
+  static final onboardingPrivateKey =
+      "5JhM4vypLzLdDtHo67TR5RtmsYm2mr8F2ugqcrCzfrMPLvo8cQW";
   static final testingPrivateKey = "";
   static final cpuAccountName = "harvst.seeds";
-  static final cpuPrivateKey = "5Hy2cvMbrusscGnusLWqYuXyM8fZ65G7DTzs4nDXyiV5wo77n9a";
+  static final cpuPrivateKey =
+      "5Hy2cvMbrusscGnusLWqYuXyM8fZ65G7DTzs4nDXyiV5wo77n9a";
   static final debugInviteSecret = "";
   static final enableHttpMock = false;
-  static final termsAndConditionsUrl = "https://www.joinseeds.com/seeds-app-terms-and-conditions.html";
-  static final privacyPolicyUrl = "https://www.joinseeds.com/seeds-app-privacy-policy.html";
+  static final termsAndConditionsUrl =
+      "https://www.joinseeds.com/seeds-app-terms-and-conditions.html";
+  static final privacyPolicyUrl =
+      "https://www.joinseeds.com/seeds-app-privacy-policy.html";
   static final explorer = "https://telos.bloks.io";
-}
\ No newline at end of file
+  static final dhoExplorer =
```

Comment:
do we need this hard coded IP address? At the very least it should be a server name.

Reply:
Actually this just a bug - looks like localhost?